blockchain/genesis/genesis.go

```diff
@@ -396,6 +396,120 @@ func (g *Genesis) Hash() hash.Hash256 {
 	return hash.Hash256b(b)
 }
 
+func (g *Genesis) isPost(targetHeight, height uint64) bool {
+	return height >= targetHeight
+}
+
+// IsPostPacific checks whether height is equal to or larger than pacific height
+func (g *Genesis) IsPostPacific(height uint64) bool {
+	return g.isPost(g.PacificBlockHeight, height)
+}
+
+// IsPrePacific checks whether height is before pacific height
+func (g *Genesis) IsPrePacific(height uint64) bool {
+	return !g.IsPostPacific(height)
+}
```

Comment:
i was thinking, rename `IsPostPacific()` to `IsPacific()` and remove `IsPre`
so it is used like
```
if g.IsPacific()
if !g.IsPacific()
```

Reply:
sounds good